class-lifterlms.php

```diff
@@ -98,8 +96,7 @@ private function __construct() {
 		 */
 		$this->localize();
 
-		// Include required files.
-		$this->includes();
+		$this->query = new LLMS_Query();
```

Comment:
I was a little worried about this, because it changed its position.
So I manually forced this line class.llms.install.php::check_version()
`		if ( ! defined( 'IFRAME_REQUEST' ) && get_option( 'lifterlms_current_version' ) !== LLMS()->version ) {`
to
`		if ( ! defined( 'IFRAME_REQUEST' ) && get_option( 'lifterlms_current_version' ) === LLMS()->version ) {`
I reloaded the admin and got this:
`[27-May-2020 17:52:45 UTC] PHP Fatal error:  Class 'LLMS_Admin_Table' not found in /srv/www/wordpress-one/public_html/wp-content/plugins/lifterlms/includes/admin/settings/tables/class.llms.table.notification.settings.php on line 21`

does this rings you any bells?
It's unrelated to the query thing of course.

Reply:
nice catch... it doesn't ring any bells but I see the issue and I'm working on it

Reply:
I think I've got this more solidly working now.